ide/db/src/org/netbeans/modules/db/explorer/dlg/CreateTableDialog.java

```diff
@@ -471,7 +471,7 @@ public DataTable(TableModel model) {
                 Map cmap = ColumnItem.getColumnProperty(i);
                 col.setIdentifier(cmap.get("name")); //NOI18N
                 columnName = NbBundle.getMessage (CreateTableDialog.class, "CreateTable_" + i); //NOI18N
-                columnWidth = (new Double(getFontMetrics(getFont()).getStringBounds(columnName, getGraphics()).getWidth())).intValue() + 20;
+                columnWidth = Double.valueOf(getFontMetrics(getFont()).getStringBounds(columnName, getGraphics()).getWidth()).intValue() + 20;
```

Comment:
i think `Double.valueOf` can be completely removed here, everything is an int already if i see this correctly.